speech-api/SpeechSynthesis-speak-SSML.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+// The shortest possible valid SSML document to speak "one":
+const ssmlText = `<speak version="1.1"
+       xmlns="http://www.w3.org/2001/10/synthesis" xml:lang="en-US">
+  one
+</speak>`;
+
+// A precondition test to ensure the SSML can be parsed.
+// The resulting document isn't used in the actual test.
+test(() => {
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(ssmlText, 'text/xml');
+  assert_equals(doc.documentElement.textContent.trim(), 'one');
+}, 'precondition: valid XML document');
+
+async_test(t => {
+  // The assumption of this test is that if SSML is *not* supported at all,
+  // then "less than speak version equals one point one ..." will be spoken,
```

Comment:
Currently [the spec says](https://w3c.github.io/speech-api/#utterance-attributes):

>For speech synthesis engines that do not support SSML, or only support certain tags, the user agent or speech engine must strip away the tags they do not support and speak the text.

I think this means browsers without SSML support should also just speak "one", assuming that the *tags* also include `<speak>`.

Reply:
That would be a form of support of SSML, really. One could pass this test with such tag-stripping support, it's true.